src/main/java/it/gov/pagopa/pu/registry/model/PagopaRegistry.java

```diff
@@ -0,0 +1,40 @@
+package it.gov.pagopa.pu.registry.model;
+
+import lombok.*;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@Document(collection = "pagopa_registry")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@EqualsAndHashCode
+@ToString
+public class PagopaRegistry implements Serializable {
+
+  @Id
+  private String registryId;
+  private String registryOrigin;
+  private String registryType;
```

Comment:
we will need this only on the DTO read from the topic, we could avoid to store it